config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
It looks like this API key might still be exposed? But I also see that you added this file to gitignore so probably fine going forward?